### PR TITLE
Update Human-in-the-loop example

### PR DESCRIPTION
**Pull request by c-bata**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

Follow-up https://github.com/optuna/optuna-dashboard/pull/428

## What does this implement/fix? Explain your changes.

Please merge this PR after Optuna v3.1.0 and Optuna Dashboard v0.10.0 are released.

### CODE REVIEW

File: docs/tutorials/hitl.rst [hunk 1]
```diff
@@ -201,20 +201,18 @@ In the ``suggest_and_generate_image`` function, a new Trial is obtained and new
 .. code-block:: python
     :linenos:
 
-    def start_optimization(
-        storage: optuna.storages.BaseStorage, artifact_backend: FileSystemBackend
-    ) -> NoReturn:
+    def start_optimization(artifact_backend: FileSystemBackend) -> NoReturn:
         # 1. Create Study
-        sampler = optuna.samplers.TPESampler(constant_liar=True)
+        storage = "sqlite:///db.sqlite3"
```
Comment (keisuke-umezawa) [inline docs/tutorials/hitl.rst:206]: [nits] It seems that we do not need this line.

File: examples/hitl/main.py [hunk 2]
```diff
@@ -73,18 +70,15 @@ def start_optimization(
...
 
 def main() -> NoReturn:
     tmp_path = os.path.join(os.path.dirname(__file__), "tmp")
 
-    # 1. Create RDBStorage
-    url = "sqlite:///db.sqlite3"
-    storage = optuna.storages.RDBStorage(url=url)
-
-    # 2. Create Artifact Storage
+    # 1. Create Artifact Store
```
Comment (keisuke-umezawa) [inline examples/hitl/main.py:81]: [question] Why did we move those line into `start_optimization`?

**Review by keisuke-umezawa — APPROVED** [review]
Just left nits and question. You can merge it anytime.

File: docs/tutorials/hitl.rst (hunk 1 above)
Comment (c-bata) [reply docs/tutorials/hitl.rst:206]: Correct! Good catch.

```suggestion
```

File: examples/hitl/main.py (hunk 2 above)
Comment (c-bata) [reply examples/hitl/main.py:81]: To just simplify the code. Thank you for your question.

**Pull request merged**